getting-started/iterate-new-app-intellij.hbs.md

```diff
@@ -0,0 +1,141 @@
+    For example, `.` uses the working directory, or you can specify a full file path.
+    3. In the **Namespace** field, optionally provide the namespace that the workload should be associated with on the cluster. If you followed the steps to [Prepare your IDE to iterate on your application](/iterate-new-app-intellij.hbs.md#prepare-to-iterate) you do not need to enter a namespace and IntelliJ will use the namespace you associated with your context.
+
+The Apply Workload command will run, which opens a terminal and shows you the output of the Workload Apply. You can also monitor your application as it's being deployed to the cluster on the `Tanzu Panel`. The `Tanzu Panel` shows the workloads in the namespace associated with your current Kubernetes context on the left side, and the details of the Kubernetes resources for the workloads running in the namespace associated with your current Kubernetes context in the center. The Apply Workload command can take a few minutes to deploy your application onto the cluster.
+
+
+## <a id="live-update-your-app"></a>Live update your application
+
+Deploy the application to view it updating live on the cluster to see how code changes behave on a production cluster.
+
+Follow the following steps to live update your application:
+
+1. Create a Run Configuration.
+   1. In IntelliJ, select the `Edit Run/Debug configurations` dropdown in the top-right corner. Alternatively, navigate to `Run` > `Edit Configurations`.
+   1. Select `Tanzu Live Update`.
+   1. Select `Add new run configuration`, or click the plus icon at the top of the list.
+   1. Give your new run configuration a name, for example `Tanzu Live Update - tanzu-java-web-app`.
+   1. In the **Tiltfile Path** field, provide the path to the `Tiltfile` in the Tanzu Java Web App project directory. Select the folder icon on the right-side of the field, navigate to the Tanzu Java Web App directory, select the `Tiltfile` and click the `Open` button. The `Tiltfile` is used to facilitate Live Update using Tilt.
+   1. In the **Local Path** field, provide the path to the directory containing the Tanzu Java Web App. The Local Path value tells the Tanzu Developer Tools extension which directory on your local file system to bring into the source image container image.
+   For example, `/Users/developer/Documents/tanzu-java-web-app`.
+   1. In the **Source Image** field, provide the destination image repository to publish an image containing your workload source code. The Source Image value tells the Tanzu Developer Tools extension where to publish the container image with your uncompiled source code, and what to name that image. The image must be published to a container registry where you have write (push) access.
+   For example, `gcr.io/myteam/tanzu-java-web-app-source`.
+
```

Comment:
question: are we missing an "apply" to save changes?

Reply:
good catch